#1/template.md

```diff
@@ -1,8 +1,9 @@
 # Project Name
+Start3
 
 ## Former Participation in Celo Hackathons
 
-- Link to hackathon submission
+- [Link to hackathon submission](https://github.com/Blssngx/START3)
 
 ## Goal for this Proof of Ship
 
```

Comment:
Hi @Blssngx ,

please provide a goal for this contest to get your PR approved.